client/src/lib/tests/Backend-test.js

```diff
@@ -156,7 +156,7 @@ describe('Backend', () => {
           'Content-Type': 'application/x-www-form-urlencoded',
         });
 
-      return promise
+      promise
```

Comment:
Jest complained that tests can either return a promise or accept a `done` argument - not both.